src/locales/en_US.ts

```diff
@@ -1516,7 +1516,10 @@ const strings = {
   rewards_card_welcome_more_info: `Learn More`,
   rewards_card_new_card_button_label: `Buy New Card`,
   rewards_card_call_to_action: `Buy Prepaid Visa® Cards`,
-  rewards_card_dashboard_expires_label: `Expires`,
+  rewards_card_dashboard_field_expires_label: `Expires`,
```

Comment:
Per eng housekeeping notes, remove `rewards_card` prefix and change key to be more generic in case these are useful elsewhere.

Reply:
ie.

```
buy_prepaid_visa_button
expires_label
purchase_asset_label
```

pls apply to all other string keys from visa program

Reply:
went with `string_expires` which is apart of another group in the strings file.

Reply:
Prefixes still not removed